modules/configuring-the-training-job.adoc

```diff
@@ -17,10 +17,10 @@ ifdef::cloud-service[]
 endif::[]
 
 ifndef::upstream[]
-* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing_distributed_workloads_cluster-admin[Managing distributed workloads].
 endif::[]
 ifdef::upstream[]
-* You have access to a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/managing-openshift-ai/#managing_distributed_workloads_cluster-admin[Managing distributed workloads].
```

Comment:
I _think_ this link should be: {odhdocshome}/managing-odh/#managing_distributed_workloads[Managing distributed workloads].